default/scripting/policies/NECESSITY.focs.txt

```diff
@@ -13,11 +13,11 @@ Policy
                 ProductionCenter
                 OwnedBy empire = Source.Owner
                 Focus type = "FOCUS_RESEARCH"
-                Not Happiness low = (NamedReal name = "PLC_NECESSITY_MAX_STABILITY" value = 6)
+                Happiness low = 0 high = (NamedReal name = "PLC_NECESSITY_MAX_STABILITY" value = 6)
```

Comment:
This will produce a zero-effect at 6, better use
```
Not Happiness low = (NamedReal name = "PLC_NECESSITY_MAX_STABILITY" value = 6)
Happiness low =0
```

Reply:
I did as you suggest.
I also added a -0.1 on the activation so that small RP changes (<0.0*) are not considered, because except for huge populations (>70), the bonus when stability is 5.9+ is too small to bother counting it.
Any objection to that hack?

Reply:
Well, if some planet actually has a stability of 5.9x and the descriptions says necessity activates below 6, I think its better to show something, even if the effect value is displayed as 0.00. In fact at exactly 5.9, a populations of 16 is enough to get 0.01 and I since the display value is rounded, it would show 0.01 even for pop 8.

Reply:
Fair enough.